README.md

```diff
@@ -117,7 +117,9 @@ Start by [adding widgets](https://community.algolia.com/instantsearch.js/v2/widg
 
 ## Browser support
 
-We support the last **two versions of major browsers**. To support [IE11](https://en.wikipedia.org/wiki/Internet_Explorer_11), we recommend using [polyfill.io](https://polyfill.io).
+We support the last **two versions of major browsers** (Chrome, Edge, Firefox, Safari).
+
+To support [IE11](https://en.wikipedia.org/wiki/Internet_Explorer_11), we recommend loading [polyfill.io](https://polyfill.io) with the following URL `https://cdn.polyfill.io/v2/polyfill.min.js?features=default,Array.prototype.includes`
```

Comment:
For the usage of `.includes` we should make a choice. Either we consider it as available (via polyfill when it's not) then we use it everywhere and we drop the usage of `lodash/includes`. Or we don't use this feature until it's widely supported and we keep using the lodash version. WDYT?

Reply:
It's about moving towards the direction we set: removing lodash at the end. We're in an intermediary step where we have some parts that will include both, and this PR fixes the state for end-users who do not have the latest version of browsers.

Reply:
It's removed! :)